src/core/attributes.js

```diff
@@ -242,7 +242,7 @@ p5.prototype.strokeCap = function(cap) {
  *
  * @method strokeJoin
  * @param  {Constant} join either MITER, BEVEL, ROUND
- * @return {p5}                   the p5 object
+ * @chainable
```

Comment:
indeed, and it gives the opportunity later to customize the render specifically for this case.